gno.land/pkg/gnoland/app.go

```diff
@@ -34,11 +34,12 @@ import (
 
 // AppOptions contains the options to create the gno.land ABCI application.
 type AppOptions struct {
-	DB                dbm.DB             // required
-	Logger            *slog.Logger       // required
-	EventSwitch       events.EventSwitch // required
-	VMOutput          io.Writer          // optional
-	InitChainerConfig                    // options related to InitChainer
+	DB                 dbm.DB             // required
+	Logger             *slog.Logger       // required
+	EventSwitch        events.EventSwitch // required
+	VMOutput           io.Writer          // optional
+	InitChainerConfig                     // options related to InitChainer
+	ParamsMaxCacheSize int                // optional
```

Comment:
What is the int representing? Kb? Mb? The number of items?

Reply:
```suggestion
	ParamsMaxCacheSize int                // optional, number of items
```